docs/_parser.lua

```diff
@@ -0,0 +1,196 @@
+local gio = require("lgi").Gio
```

Comment:
Could you move the addition of this file and the new `COPY` for it (why is that needed?) to the next commit? That way, it doesn't look like just new code and one can actually see which part was just moved and which is actually new.

Reply:
> why is that needed?

To avoid messing with LUA_PATH. I can use either solution, I don't mind.